### PR TITLE
Function improvements

### PR DESCRIPTION
**Pull request by jpquast**

### CODE REVIEW

File: R/calculate_go_enrichment.R [hunk 1]
```diff
@@ -35,6 +35,13 @@ go_enrichment <- function(...) {
 #' corresponding protein has a significantly changing peptide. The input data frame may contain
 #' peptide level information with significance information. The function is able to extract
 #' protein level information from this.
+#' @param group a character column in the \code{data} data frame that contains information by
+#' which the analysis should be grouped. The analysis will be performed seperately for each of the
+#' groups. This is most likely a column that lables seperate comparisons of different conditions.
```
Comment (dschust-r) [inline R/calculate_go_enrichment.R:40]:
```suggestion
#' groups. This is most likely a column that labels separate comparisons of different conditions.
```

File: R/calculate_go_enrichment.R [hunk 2]
```diff
@@ -35,6 +35,13 @@ go_enrichment <- function(...) {
 #' corresponding protein has a significantly changing peptide. The input data frame may contain
 #' peptide level information with significance information. The function is able to extract
 #' protein level information from this.
+#' @param group a character column in the \code{data} data frame that contains information by
+#' which the analysis should be grouped. The analysis will be performed seperately for each of the
```
Comment (dschust-r) [inline R/calculate_go_enrichment.R:39]:
```suggestion
#' which the analysis should be grouped. The analysis will be performed separately for each of the
```

**Review by dschust-r — CHANGES_REQUESTED** [review]
Found small typos but otherwise LGTM 👍🏻

File: R/calculate_go_enrichment.R (hunk 1 above)
Comment (jpquast) [reply R/calculate_go_enrichment.R:40]: Will change it in future PR.

File: R/calculate_go_enrichment.R (hunk 2 above)
Comment (jpquast) [reply R/calculate_go_enrichment.R:39]: Will change it in future PR.

**Pull request merged**